metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/iceberg/IcebergTableHandler.java

```diff
@@ -207,11 +218,11 @@ public boolean update(final TableInfo tableInfo) {
             }
             if (result) {
                 updateSchema.commit();
-                final String newTableMetadataLocation = icebergMetastoreTables.getTableOps().currentMetadataLocation();
```

Comment:
Use the method currentMetadataLocation. The method needs to be overwritten in our extended class below.

Reply:
that's the changing I am working on. will update

Reply:
Added back this method.